### PR TITLE
[feat][doc] Add docs for Snowflake sink connector

### PR DESCRIPTION
**Pull request by danpi**
### Motivation
Add snowflake sink connector docs to SN Hub.

### Modification

Add docs for the Snowflake sink connector:

- Affected releases: 2.9.4; 2.10.3
- Add a YAML file to sync docs between Snowflake repo to SN Hub
- Add related images

### Documentation preview link
https://staging--streamnative-hub.netlify.app/connectors/snowflake-sink/2.9.4

### CODE REVIEW

**Comment by danpi** [comment]
@Huanli-Meng PTAL, thanks. Need your help to provide `snowflake-logo.png.`

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+  autoAck: false
+```
+
+# How to use
+
+You can use the Snowflake sink connector with Function Worker or Function Mesh.
+
+## Work with Function Worker
+
+You can use the Snowflake sink connector as a non built-in connector or a built-in connector as below.
```
Comment (momo-jun) [inline connectors/snowflake-sink/2.10.3/snowflake-sink.md:241]: "non built-in" sounds a bit weird to me. Can we use "standalone" or "freestanding" or something else instead?

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+owner_img: "/images/streamnative.png"
+id: "snowflake-sink"
+---
+
+The [Snowflake](https://www.snowflake.com/) sink connector pulls data from Pulsar topics and persists data to Snowflake.
+
+![](/images/connectors/snowflake-sink.png)
+
+# Features
+This section describes features of the Snowflake sink connector. For details about how to configure these features, see [how to configure](#how-to-configure).
```
Comment (momo-jun) [inline connectors/snowflake-sink/2.10.3/snowflake-sink.md:29]:
```suggestion
This section describes the features of the Snowflake sink connector. For details about how to configure these features, see [how to configure](#how-to-configure).
```

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+| `warehouse`                | String  | No       | ""(empty string)                                        | The warehouse name in the snowflake. By default, no warehouse name is set.                                                                                                                                                                                                                                                                                     |
+| `bufferCountRecords`       | int     | No       | 10_000                                                  | The number of records that are buffered in the memory before they are ingested to Snowflake. By default, it is set to `10_000`.                                                                                                                                                                                                                                |
```
Comment (momo-jun) [inline connectors/snowflake-sink/2.10.3/snowflake-sink.md:133]:
```suggestion
| `bufferCountRecords`       | int     | No       | 10_000                                                  | The number of records that are buffered in the memory before they are ingested by Snowflake. By default, it is set to `10_000`.                                                                                                                                                                                                                                |
```

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+This example shows how to create a Snowflake sink connector on a Pulsar cluster using the [`pulsar-admin sinks create`](https://pulsar.apache.org/tools/pulsar-admin/2.10.0-SNAPSHOT/#-em-create-em--30) command.
+
+```
+PULSAR_HOME/bin/pulsar-admin sinks create \
+--sink-config-file <snowflake-sink-config.yaml >
+```
+
+### Use it as built-in connector
+
+You can make the Snowflake sink connector as a built-in connector and use it on a standalone cluster.
```
Comment (momo-jun) [inline connectors/snowflake-sink/2.10.3/snowflake-sink.md:256]:
```suggestion
You can make the Snowflake sink connector as a built-in one and use it on a standalone cluster.
```

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md (hunk 3 above)
Comment (danpi) [reply connectors/snowflake-sink/2.10.3/snowflake-sink.md:133]: The idea here is to insert into the Snowflake, not insert through the Snowflake.

File: connectors/snowflake-sink/2.10.3/snowflake-sink.md (hunk 1 above)
Comment (danpi) [reply connectors/snowflake-sink/2.10.3/snowflake-sink.md:241]: Replace it by `standalone `.

**Comment by danpi** [comment]
@momo-jun I have applied your suggestions. PTAL, thanks.

**Comment by Huanli-Meng** [comment]
> @momo-jun I have applied your suggestions. PTAL, thanks.

@danpi could you also please help apply these updates to the connector repo doc? Because, when the connector is released, docs are still synced from the connector repo doc. Thanks.

**Comment by danpi** [comment]
> > @momo-jun I have applied your suggestions. PTAL, thanks.
> 
> @danpi could you also please help apply these updates to the connector repo doc? Because, when the connector is released, docs are still synced from the connector repo doc. Thanks.

Sure, I will sync them.

**Pull request merged**